src/transport/raw/MessageHeader.h

```diff
@@ -50,6 +50,13 @@ typedef int PacketHeaderFlags;
 
 namespace Header {
 
+enum Consts
+{
+    kHeaderMinLength        = 8,
+    kPrivacyHeaderMinLength = 4,
+    kPrivacyHeaderOffset    = 4,
```

Comment:
These should be under PacketHeader, no?  They have nothing to do with PayloadHeader....

Reply:
Moved